internal/binutils/binutils.go

```diff
@@ -144,7 +144,10 @@ func initTools(b *binrep, config string) {
 		// that if the tool is not found by its default name.
 		b.addr2line, b.addr2lineFound = findExe("gaddr2line", append(paths["addr2line"], defaultPath...))
 	}
-	b.nm, b.nmFound = findExe("nm", append(paths["nm"], defaultPath...))
+	// The "-n" option is supported by llvm since 2011. The output of llvm-nm
```

Comment:
nit: llvm -> LLVM

Always spell initialisms with all caps, unless it's a name of a command line tool (`llvm-nm`).

Reply:
Done.